OpenRA.Game/FileFormats/AudLoader.cs

```diff
@@ -153,7 +166,33 @@ public static byte[] LoadSound(Stream s)
 				dataSize -= 8 + chunk.CompressedSize;
 			}
 
-			return output;
+			rawData = output;
+			return true;
+		}
+
+		public bool TryParseSound(Stream stream, string fileName, out byte[] rawData, out int channels, out int sampleBits,
+			out int sampleRate)
+		{
+			channels = sampleBits = sampleRate = 0;
+
+			try
+			{
+				if (!LoadSound(stream, out rawData))
+					return false;
+			}
+			catch (Exception e)
+			{
+				Log.Write("debug", "Failed to parse AUD file {0}. Error message:".F(fileName));
```

Comment:
If you want to keep the current interface/api you will need to keep the try/catch, but remove the logging.